Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "polyline"
 description = "Encoder and decoder for the Google Encoded Polyline format"
-version = "0.9.0"
-authors = ["Tom MacWright <tom@macwright.org>", "The GeoRust Developers"]
+version = "0.10.0"
```

Comment:
This change doesn't break any api's - so there's no reason for a semver breaking change, right?

It'd be nice if existing users could just get this perf bonus "for free" without having to bump their deps.

Reply:
Yes we could publish 0.9.1…

Reply:
I'll follow up with a PR.

Reply:
haha well, I was wrong, it *is* a breaking release after all: See #31